cobalt/browser/h5vcc_metrics/public/mojom/h5vcc_metrics.mojom

```diff
@@ -0,0 +1,28 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+module h5vcc_metrics.mojom;
+
+interface MetricsListener {
+  OnMetrics(string tbd);
+};
+
+interface H5vccMetrics {
+  AddListener(pending_remote<MetricsListener> listener) => ();
```

Comment:
This could be my ignorance, but how does this map to the current `onMetricEvent` in H5vccMetrics?

Reply:
Ah, I forgot to tie this in from `H5vccMetrics::setOnmetrics()` (had it but must have removed it).

Reply:
Should be fixed now.